src/satosa/base.py

```diff
@@ -134,6 +134,12 @@ def _auth_resp_callback_func(self, context, internal_response):
         """
 
         context.request = None
+        context_state = context.state.get(STATE_KEY)
+        if not context_state:
+            redirect_url = self.config.get("UNKNOW_ERROR_REDIRECT_PAGE")
```

Comment:
This needs to be removed as @ioparaskev mentioned.

Reply:
@ioparaskev @c00kiemon5ter I miss something here, should we really print the things only on the logs and do not redirect users to a informational page? 

We could find a transitional solution if you agree, something good for our users and then a following refactor with more design elements in the hands. Let me know, you know my limits, try your best :)